### PR TITLE
[BI-34] Create UI for Forgot Password

### PR DESCRIPTION
**Pull request by alexstefan-binarcode**
Created a route and a view for the forgot password functionality. Also added a translation.

### CODE REVIEW

File: src/views/ForgotPassword.vue [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+      </h2>
+    </div>
+    <div class="mt-8 sm:mx-auto sm:w-full sm:max-w-md">
+      <div class="px-4 py-8 bg-white shadow sm:rounded-lg sm:px-10">
+        <ValidationObserver v-slot="{ valid }" ref="form">
+          <form @submit.prevent="handleSubmit">
+            <base-input
+                v-model="model.email"
+                type="email"
+                name="email"
```
Comment (Ceciliaa) [inline src/views/ForgotPassword.vue:16]: missing translation

**Review by Ceciliaa — APPROVED** [review]

File: src/views/ForgotPassword.vue [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  data() {
+    return {
+      model: {
+        email: ''
+      },
+    };
+  },
+  methods: {
+    handleSubmit() {
+      // I took advantage of the validate() method (on form) provided by Vee-Validate, which
```
Comment (PascuMihai) [inline src/views/ForgotPassword.vue:48]: this comment can be removed

File: src/views/ForgotPassword.vue [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+      // checks the validated state of the inputs.
+      // validate() returns a promise.
+      // https://vee-validate.logaretm.com/v2/guide/components/validation-observer.html
+
+      this.$refs.form.validate()
```
Comment (PascuMihai) [inline src/views/ForgotPassword.vue:53]: A better approach would be this 
```
const valid = await this.$refs.loginObserver.validate();
        if (!valid) {
          return
        }
```

File: src/views/ForgotPassword.vue [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+        .then((result) => {
```
Comment (PascuMihai) [inline src/views/ForgotPassword.vue:54]: And then here just log form valid

**Review by PascuMihai — CHANGES_REQUESTED** [review]
Fix comments

File: src/views/ForgotPassword.vue [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+        email: ''
+      },
+    };
+  },
+  methods: {
+    async handleSubmit() {
+      const valid = await this.$refs.form.validate()
+
+      if (!valid) return;
+      else console.log("Form valid");
```
Comment (PascuMihai) [inline src/views/ForgotPassword.vue:51]: There is no need for else here, if the form is valid will just skip the if condition anyway

**Review by PascuMihai — APPROVED** [review]

**Pull request merged**